apps/cli/src/commands/boot-challenger.ts

```diff
@@ -71,7 +70,18 @@ const initCli = async () => {
         throw new Error("No private key passed in. Please provide a valid private key.")
     }
 
-    const { address, signer } = getSignerFromPrivateKey(walletKey);
+    let address: string;
+    let signer: Signer;
+    
+    try {
+        const signerResponse = getSignerFromPrivateKey(walletKey).signer;
+        address = await signerResponse.getAddress();
+        signer = signerResponse;
+    } catch (error) {
+        console.error(`Error getting signer from private key: ${(error as Error).message}`);
+        return;
+    }
```

Comment:
Why is this added here ?

Reply:
Because if you provide an invalid private key, the CLI crashes without it.